index.html

```diff
@@ -0,0 +1,931 @@
+                <div class="adjustMargin">
+                <h2>
+                <a href="#" class="noTextDecoration ">
+                    <b class="textLatestNews">Coronavirus is growing exponentially — here’s what the numbers mean </b>
+                    <br/>
+                </a>
+                </h2>
+                <a href="#" class="noTextDecoration authorName"> The Conversation </a> <span class ="timeAgo">6 hours ago</span>
+                </div>
+            </article>
+            <article class="newsArticle6">
+                <a href="#"><img src="https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2020/04/Untitled-design51-516x270.png" class="article1Image" alt="News Article 6 Image"></a>
+                <div class="adjustMargin">
+                <h2>
+                <a href="#" class="noTextDecoration ">
+                    <b class="textLatestNews">AI reveals that mice's faces express a range of emotions -- just like humans </b>
+                    <br/>
+                </a>
+                </h2>
+                <a href="#" class="noTextDecoration authorName"> Thomas Macaulay </a> <span class ="timeAgo">4 hours ago</span>
+                </div>
+            </article>
+            <article class="newsArticle7">
+                <a href="#"><img src="https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2020/04/Getty-Photo-Challenge-516x270.jpg" class="article1Image" alt="News Article 7 Image"></a>
+                <div class="adjustMargin">
+                <h2>
+                <a href="#" class="noTextDecoration ">
+                    <b class="textLatestNews">Life imitating art: Quarantined people remix famous paintings with household items  </b>
+                    <br/>
+                </a>
+                </h2>
+                <a href="#" class="noTextDecoration authorName"> Ivan Mehta </a> <span class ="timeAgo">8 hours ago</span>
+                </div>
+            </article>
+            <article class="newsArticle8">
+                <a href="#"><img src="https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2020/04/zoombombing-513x270.png" class="article1Image" alt="News Article 8 Image"></a>
+                <div class="adjustMargin">
+                <h2>
+                <a href="#" class="noTextDecoration ">
+                    <b class="textLatestNews">NYC classrooms cancel Zoom after trolls make 'Zoombombing' a thing  </b>
+                    <br/>
+                </a>
+                </h2>
+                <a href="#" class="noTextDecoration authorName"> TNW Deals </a> <span class ="timeAgo">7 hours ago</span>
+                </div>
+            </article>
+        </section>
+        <section class="latestFunding">
+            <div class="latestFundingHeader">
+                <a href="#" class="redHeading noTextDecoration">
+                    Latest funding rounds on Index.co >    
```

Comment:
The title and the small menu are more separated from the section than they should. Please adjust the spacing.
This section is wider than it should be, too.